src/bundle/upload.ts

```diff
@@ -68,6 +70,14 @@ export const uploadBundle = async (appid: string, options: Options, shouldExit =
     p.log.error(`Path ${path} does not exist, build your app first, or provide a valid path`);
     program.error('');
   }
+
+  const isPluginConfigured = searchInDirectory(path, 'notifyAppReady')
+
+  if (!isPluginConfigured) {
+    p.log.error(`Plugin is not configured in your JavaScript code see https://capgo.app/docs/plugin/api/#notifyappready`);
```

Comment:
I think the error message could be improved. Seeing this error I would believe I did not include the plugin correctly using capacitor. But the plugin is correctly configured only the usage is wrong. Something in the lines of "Did not find a call to notifyAppReady in the source code. Please make sure to call it to properly set up updates (see link)." would push the developer more in the right direction.